src/nni_manager/rest_server/nniRestServer.ts

```diff
@@ -34,7 +34,8 @@ export class NNIRestServer extends RestServer {
      */
     protected registerRestHandler(): void {
         this.app.use(express.static('static'));
-        this.app.use(bodyParser.json());
+        this.app.use(bodyParser.json({limit: '50mb'}));
+        this.app.use(bodyParser.urlencoded({limit: '50mb', extended: true}));
```

Comment:
I'm not familiar with body parser. Does this mean the URL can be 50mb long?

Reply:
It means the data of  sending post request can be 50mb long.
```python
response = rest_post(experiment_url(port), json.dumps(request_data), REST_TIME_OUT, show_error=True)
def rest_post(url, data, timeout, show_error=False):
    '''Call rest post method'''
    try:
        response = requests.post(url, headers={'Accept': 'application/json', 'Content-Type': 'application/json'},\
                                 data=data, timeout=timeout)
        return response
    except Exception as exception:
        if show_error:
            print_error(exception)
        return None
```

Reply:
The search space(json file) will be so big(>100kb), when you use BatchTunner. If not increase the limit of request body, it will fail for HTTP:413 Request Entity Too Large
```python
def set_experiment(experiment_config, mode, port, config_file_name):
    '''Call startExperiment (rest POST /experiment) with yaml file content'''
    ......
    request_data['searchSpace'] = experiment_config.get('searchSpace')
    ......
    response = rest_post(experiment_url(port), json.dumps(request_data), REST_TIME_OUT, show_error=True)

```

Reply:
@Sundrops , thanks for the fix. One question: does this problem exist if you only keep line37 and remove line38?

Reply:
Only keeping line37 can fix this problem now. I add line38 for the future. Do I need to remove line38?

Reply:
@Sundrops thanks, I think we can only keep line37 for now :)

Reply:
@QuanluZhang Done.